entrypoint.sh

```diff
@@ -18,6 +18,16 @@ sh -c "git config --global credential.username $GITLAB_USERNAME"
 sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
+if [[ -n "${REMOVE_BRANCH}" ]]; then # Check if variable exists 
+   if [[ "${REMOVE_BRANCH}" == "true" ]]; then 
+      # If branch exists; w/o sh -c ""
+      branchExists=$(git ls-remote $(git remote get-url --push mirror) ${CHECKOUT_BRANCH:-$DEFAULT_GITHUB_REF} | wc -l)
+      if [[ ${branchExists} == 1 ]]; then
+         sh -c "echo removing $branch branch at $(git remote get-url --push mirror)"		
```

Comment:
Does this work? Seems like the message should be quoted somehow. Is the echo just gobbling up everything that comes after it? I haven't tested this, but I would think this line should be more like:
```
sh -c "echo \"removing the branch ${branch} at $(git remote get-url --push mirror)\""
```

Also, is the `sh -c` really necessary? Aren't you already in a bash shell?

Reply:
It works but it is cleaner your way
I will remove the sh -c